opentera-webrtc-native-client/CMakeLists.txt

```diff
@@ -23,8 +23,28 @@ option(OPENTERA_WEBRTC_USE_SYSTEM_OPENCV "Use system OpenCV library for linking"
 # Default behavior is to enable tests
 option(OPENTERA_WEBRTC_ENABLE_TESTS "Build tests" ON)
 
+# Default behavior is to enable HTML documentation
 option(OPENTERA_WEBRTC_ENABLE_PYTHON_HTML_DOC "Generate HTML documentation" ON)
 
+# Default behavior is to install only on "install" target
+option(
+    OPENTERA_WEBRTC_NATIVE_CLIENT_PYTHON_INSTALL_ON_BUILD
+    "If ON, install the native client python library on build instead of install"
+    OFF
+    )
+
+# Default behavior is to install to --user in both cases
+set(
+    OPENTERA_WEBRTC_NATIVE_CLIENT_PYTHON_PIP_DEVEL_OPTIONS
+    "--user"
+    CACHE STRING
+    "Pip install options when building"
+)
+set(OPENTERA_WEBRTC_NATIVE_CLIENT_PYTHON_PIP_INSTALL_OPTIONS
+    "--user"
+    CACHE STRING
+    "Pip install options when installing"
+)
```

Comment:
Added these options that allows catkin to install to devel or install space using `--target=[path]` pip argument